components/usage/pkg/controller/reconciler_test.go

```diff
@@ -202,7 +201,6 @@ func TestUsageReportConversionToDBUsageRecords(t *testing.T) {
 				StartedAt:     creationTime.Time(),
 				StoppedAt:     sql.NullTime{},
 				CreditsUsed:   470,
-				GenerationId:  0,
```

Comment:
Why remove `GenerationId` here and on line 183?

Reply:
0 is the default value so it doesn't need to be set when constructing the object.